test/api_test.rb

```diff
@@ -6,12 +6,12 @@ def setup
     @url = 'http://my.prestashop.com'
     @url_regex = %r{my[.]prestashop[.]com.*}
     @key = 'VALID_KEY'
-  end
 
-  def test_get_resources
     xml = File.open(File.join __dir__, 'xml', 'resources.xml').read
     stub_request(:any, @url_regex).to_return body: xml
+  end
 
+  def test_get_resources
```

Comment:
I now see that this works differently than the original documentation, but matches the functionality of the code before my pull request. I can revert the test changes and fix the code if this preferable, let me know.